src/platform/forms/save-in-progress/FormSaved.jsx

```diff
@@ -53,7 +55,9 @@ class FormSaved extends React.Component {
       <div>
         <div className="usa-alert usa-alert-info">
           <div className="usa-alert-body">
-            <strong>{savedMessage(this.props.route.formConfig)}</strong>
+            <h2 className="vads-u-margin-y--0">
```

Comment:
With this going from inheriting the font size and style from the `usa-alert-body` class to an h2, do you need to add any classes to adjust font size for consistent appearance?

Reply:
Good call, I'll change the size to an H3